### PR TITLE
Update Liebert.yaml to include GXT5 series  UPS

### PR DESCRIPTION
**Pull request by thewifininja**
Please give a short description what your pull request is for
- I added an OID to the Liebert.yaml to support GXT5 series that were misidentified as Linux servers.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.
After
Before

### CODE REVIEW

**Comment by thewifininja** [comment]
I fixed the tab indention in this YAML. It should pass now.

**Comment by thewifininja** [comment]
Made an edit to the YAML. Not sure why it fails 3 tests.

**Comment by Jellyfrog** [comment]
you can just combine them to a list of oids under `sysObjectID`, no need to write it twice

File: includes/definitions/liebert.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ discovery:
     -
         sysObjectID:
             - .1.3.6.1.4.1.476.1.42
+            - .1.3.6.1.4.1.8072.3.2.10
```
Comment (murrant) [inline includes/definitions/liebert.yaml:12]: This is the standard linux sysObjectID... Looks like you might be able to use sysDescr instead at least.

**Comment by Jellyfrog** [comment]
First of all, thank you for your contribution!

We're closing this due to inactivity, feel free to reopen if you find the time to continue on this!

**Pull request closed without merging**